website/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+// prettier-ignore
+module.exports = {
+  plugins: ['react'],
+  extends: ['uber-jsx', 'uber-es2015', 'prettier', 'prettier/react', 'plugin:import/errors'],
+  overrides: {
+    files: ['*.spec.js', 'webpack.config.js', '**/bundle/*.js'],
+    rules: {
+      'import/no-extraneous-dependencies': 0
```

Comment:
Why do we need this here

Reply:
There are errors mentioning `import/no-extraneous-dependencies`
```
Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .

~/luma.gl/website/webpack.config.js
  2:17  error  'webpack' should be listed in the project's dependencies, not devDependencies                                 import/no-extraneous-dependencies
  7:17  error  'ocular-dev-tools' should be listed in the project's dependencies. Run 'npm i -S ocular-dev-tools' to add it  import/no-extraneous-dependencies

✖ 2 problems (2 errors, 0 warnings)

```

Reply:
These are irrelevant. The website uses root dev dependencies. This lint rule is for validating package.jsons of published modules.